README.md

```diff
@@ -311,11 +322,11 @@ Chainlink Functions can be used with [Chainlink Automation](https://docs.chain.l
    - Once created be sure to ensure the Upkeep has sufficient funds in it. You can add funds in the UI.
    - Find further documentation for working with Chainlink Automation here: [https://docs.chain.link/chainlink-automation/introduction](https://docs.chain.link/chainlink-automation/introduction)
 
-Once the contract is registered for upkeep, check the latest response or error with the commands `npx hardhat functions-read --network network_name_here --contract contract_address_here`. Note that you can also pause and unpause your Automation Upkeep using the web app's UI.
+Once the contract is registered for upkeep, check the latest response or error with the commands `npx hardhat functions-read --network network_name_here --contract 0x_contract_address`. Note that you can also pause and unpause your Automation Upkeep using the web app's UI.
 
 5. For debugging on your machine, use the command `npx hardhat functions-check-upkeep --network network_name_here --contract contract_address_here` to see if Automation needs to call `performUpkeep`. If this call returns `false` then the upkeep interval has not yet passed and `performUpkeep` will not execute. In order to test that `performUpkeep` will run correctly before registering the Automation upkeep, you can also trigger a request manually using the command use the command `npx hardhat functions-perform-upkeep --network network_name_here --contract contract_address_here`.
 
-You can also attach a listener to a Subscription ID by updating the `subId` variable in `/scripts/listen.js`, and then running `npm run listen --network your_network_name` from the repo root. To do this open a new terminal or split terminal and run `npm run listen`. This script uses nodemon which restarts the script when you save files or when the listener returns a result.
+You can also attach a listener to a Subscription ID by updating the `subId` variable in `/scripts/listen.js`, and then running `npm run listen --network your_network_name` from the repo root. **Note** Do this in a new terminal or split terminal so that it can keep listening as you develop. This script uses nodemon which restarts the script when you save files or when the listener returns a result.
```

Comment:
We should test this too. IIRC I might have noticed an issue here but didn't have time to re-test

Reply:
works as at my current version which is one behind the lint fix pushed in 4e165605ba124c398d8b99d2d9a95e6d2fc39b8f